influxql/parser_test.go

```diff
@@ -2812,3 +2806,11 @@ func mustParseDuration(s string) time.Duration {
 	}
 	return d
 }
+
+func Duration(v time.Duration) *time.Duration {
```

Comment:
Can you keep golint happy with some comments, or maybe even better move these to `pkg` or somewhere? Or alternatively make them unexported?